tests/test_mongoengine.py

```diff
@@ -269,7 +270,8 @@ def log_optional_url(url: str | None) -> None:
 
     post = Post.objects().get()
     print(post._id)
-    post.attachments[-1]
+    attachment = post.attachments[-1]
+    assert_type(attachment, PostAttachment)
```

Comment:
@sbdchd This PR turned out bigger than i expected, and i want to make sure all my changes are doung what's supposed to do, so i plan to add some test.

These tests is only for verifying type correctness, right?

I plan to use `assert_type` here, instead of trying to access expected object properties, and want to make sure this is correct approach.

Reply:
Yeah all the tests are only for types!